### DISCUSSION TITLE
zh-cn/posts/2024/07/05/fried-egg-recipe/

### DISCUSSION THREAD

**Discussion by giscus** [CyrusYip/blog-comments#77] [Comments]
# zh-cn/posts/2024/07/05/fried-egg-recipe/

材料与工具 # 冷藏的鸡蛋（1 个约 51g） 电磁炉 精铁平底炒锅（已开锅） 花生油（不要使用菜籽油这种低闪点的油，热锅后倒下去可能会着火） 锅铲 筷子 计时器 两个碗（一个装鸡蛋，另一个装少量水） 第一次煎蛋的流程 # 打蛋

https://cyrusyip.org/zh-cn/posts/2024/07/05/fried-egg-recipe/

**Comment by earfanfan** [comment]
如果煎荷包蛋是你的第一篇入门菜谱的话，其实我至今煎荷包蛋都是失败多过成功的。因为我租的那个房子里，厨房里是燃气灶，开火就是拧一个开关，一般我区分火力大小就是看火苗的形状，在掌握火候这方面基本就是凭感觉。

>花生油（不要使用菜籽油这种低闪点的油，热锅后倒下去可能会着火）

看到这里，我想到那个渐渐销声匿迹的食用油混装的新闻了。不过在此之前我确实没想过区分用花生油还是菜籽油，说起来我还真不知道现在厨房里放的油是哪种。

**Reply by CyrusYip** [reply]
煎荷包蛋确实有点难度，我自己少说也煎了七八十次。

你用的什么锅？我在朋友家[用燃气加铁锅煎荷包蛋](https://mastodon.social/@cyrusyip/112338483200590958)也没成功过，我觉得是火太大导致粘锅。燃气炉不像电磁炉那样能显示功率，所以很难判断火候，我能想到的唯一方法就是温度枪（我没用过）。

最近我也用燃气加不粘锅，这个组合很容易成功。电磁炉加不粘锅之前也用过，也是容易成功。

---

油只要用适合炒菜（不因高温而点燃）的就可以了。

**Reply by earfanfan** [reply]
>你用的什么锅？

我只知道我的锅是个平底锅，至于是不是不粘锅就不清楚了，是2018年刚来武汉的时候一个朋友送的她家的旧锅子。因为一直也能用，就没想过买新的，就算是不粘锅应该也早就变粘锅了吧。我一般煎荷包蛋都是失败，好在自己不介意，照吃不误，就没想过怎样才能成功，一直就是吃炒蛋了。

从现实生活的角度来说，如果你会煎荷包蛋的话，就不只会煎荷包蛋，肯定也会做别的。我很期待看你写点油炸食物的菜谱，因为我小时候被烫伤过，所以对自己去尝试油炸食物心理上是千不敢万不敢的。一般网络上的菜谱在火候这点上都是模糊的，除了空气炸锅菜谱会明确指定多少度多少分钟。

**Reply by CyrusYip** [reply]
搞清楚锅的材料很重要，因为不同锅的特点和用法是不一样的。不粘锅涂层脱落就不好用了，得买新的。

我比较拿得出手的菜大概是蒜末鸡蛋（煎或者炒）和卤鸡脚。我学习做菜进步很慢，我常常去钻研细节。一开始先体验不同的刀，包括中式切片刀、主厨刀（chef's knife）、三德刀（santoku），最后选择主厨刀。然后又去学磨刀和切菜，现在菜切得不错，磨刀还不够好。

我用了超过一年不粘锅加电磁炉，一度怀疑炒菜不够好吃是锅的问题，就换成精铁锅了。铁锅确实炒菜更好吃，不过用起来比不粘锅难，钻研开锅也花了不少时间。最后我才想起来我要学做菜，哈哈。

我对油炸没有兴趣，所以就应该写不了。关于怕烫伤的问题，要不试试用厚长的手套和长的夹子？

写一篇可复现的菜谱好难，就写这篇我已经煎了无数个鸡蛋了，以后可能就写一些简单的。
